tool/autoqa/lib/bert-canonical-annotator.py

```diff
@@ -212,6 +213,10 @@ def predict_one(self, table, arg, query, word, k):
         :param k: number of top candidates to return, this defaults to self.k if absent
         :return: a array in length k of predicted tokens
         """
+        # skip uncommon word
+        if not wordnet.synsets(word):
```

Comment:
Is it worth to bring in all of nltk and the wordnet dataset just to learn that a word is common or not? Seems like an overkill...

Reply:
I searched around and didn't see a better solution... Any suggestion?

Reply:
We make our own vocabulary file (perhaps with wordnet as the starting point) and just load that. A single vocabulary file will be a lot smaller than nltk + wordnet.